openapi-spec.json

```diff
@@ -102,17 +102,114 @@
     },
     "components": {
         "schemas": {
-            "User": {
+            "Profile": {
                 "type": "object",
                 "properties": {
-                    "firstName": {
+                    "name": {
+                        "type": "string",
+                        "required": true
+                    },
+                    "discordId": {
                         "type": "string"
                     },
-                    "lastName": {
+                    "homePageUrl": {
                         "type": "string"
                     },
-                    "age": {
-                        "type": "number",
+                    "activeTimeStart": {
+                        "Type": "int",
+                        "format": "int32"
+                    },
+                    "activeTimeEnd": {
+                        "Type": "int",
+                        "format": "int32"
+                    },
+                    "activeWeekDay": {
+                        "Type": "string",
+                        "enum": ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+                    },
+                    "mainGame": {
+                        "Type": "string"
+                    },
+                    "profileDescription": {
+                        "Type": "string"
+                    },
+                    "rank": {
+                        "Type": "string"
+                    },
+                    "createdAt": {
+                        "Type": "timestamp"
+                    }
+                }
+            },
+            "PersonalInfo": {
+                "type": "object",
+                "properties": {
+                    "userId": {
```

Comment:
let's just make all id types into `integer` and not specify any format since `int32` will definitely overflow on the db